src/NServiceBus.AcceptanceTests/Routing/When_making_endpoint_uniquely_addressable.cs

```diff
@@ -10,17 +10,18 @@
     using NServiceBus.Routing;
     using NUnit.Framework;
 
-    public class When_using_instance_ids : NServiceBusAcceptanceTest
+    public class When_making_endpoint_uniquely_addressable : NServiceBusAcceptanceTest
```

Comment:
The namespace of this test should be changed to `Routing` instead of `ScaleOut`.